unlock-app/src/components/interface/locks/CheckoutUrl/elements/LocksForm.tsx

```diff
@@ -496,16 +548,7 @@ export const LocksForm = ({
                             <MetadataList />
                           ) : (
                             <div className="grid items-center grid-cols-1 gap-2 mt-2 rounded-xl">
-                              <DynamicForm
```

Comment:
Do we still use that in other places? Would it be hard to move the validation in this one?

Reply:
Yes, I need to think a bit more about refactoring this down to more manageable parts.